packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -297,6 +298,30 @@ class DateTimePicker extends React.Component {
       const previousDateTime = this.state.dateTime ? this.state.dateTime.clone() : DateTimeUtils.createSafeDate(formattedDate, this.initialTimeZone);
       updatedDateTime = DateTimeUtils.syncDateTime(previousDateTime, date, this.timeValue, this.props.showSeconds);
 
+      if (previousDateTime.isDST() && previousDateTime.hours() === 3) {
+        let flag = false;
+        if (!updatedDateTime.isDST()) {
+          updatedDateTime.subtract(1, 'hours');
+          flag = true;
+        }
+
+        if (!previousDateTime.subtract(1, 'days').isDST() && updatedDateTime.isDST()) {
+          updatedDateTime.subtract(1, 'hours');
+          flag = true;
+        }
+
+        // eslint-disable-next-line no-underscore-dangle
+        if (previousDateTime && previousDateTime._a[3] === 3 && flag) {
+          updatedDateTime.add(1, 'hours');
+          flag = false;
+        }
+      }
+
+      // eslint-disable-next-line no-underscore-dangle
+      if (this.prevDateTime && this.prevDateTime._a[3] === 2 && this.prevDateTime.hours() === 3) {
```

Comment:
Could we use `previousDateTime` instead of needing to declare `this.prevDateTime`?

This new logic seems confusing to follow where we potentially subtract an hour from updatedDateTime a couple times and then potentially add the hour back. Adding some code comments might help for readability.

Reply:
When clearing the date value using Backspace the time value gets cleared out entirely from the `previousDateTime` object. So `previousDateTime,hours` is 0. Hence added the new variable. I'll add comments